Panic after get of extension-apiserver-authentication configmap

The log shows it can download the extension-apiserver-authentication configmap but the app crashes with `Unable to get configmap/extension-apiserver-authentication`. Log:
```
kubectl -n cop-prometheus-adapter logs -f custom-metrics-apiserver-58945686d4-6dlsv
I0629 15:51:18.618704       1 round_trippers.go:386] curl -k -v -XGET  -H "Accept: application/json, */*" -H "User-Agent: adapter/v0.0.0 (linux/amd64) kubernetes/$Format" -H "Authorization: Bearer redacted" 'https://172.23.0.1:443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication'
I0629 15:51:18.631320       1 round_trippers.go:405] GET https://172.23.0.1:443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication 200 OK in 12 milliseconds
I0629 15:51:18.631338       1 round_trippers.go:411] Response Headers:
I0629 15:51:18.631343       1 round_trippers.go:414]     Content-Type: application/json
I0629 15:51:18.631347       1 round_trippers.go:414]     Date: Fri, 29 Jun 2018 15:51:18 GMT
I0629 15:51:18.632449       1 request.go:897] Response Body: {"kind":"ConfigMap","apiVersion":"v1","metadata":{"name":"extension-apiserver-authentication","namespace":"kube-system","selfLink":"/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication","uid":"96d21401-4609-11e7-9977-005056b22938","resourceVersion":"87015391","creationTimestamp":"2017-05-31T14:01:05Z"},"data":{"client-ca-file":"-----BEGIN CERTIFICATE redacted
requestheader-group-headers":"[\"X-Remote-Group\"]","requestheader-username-headers":"[\"X-Remote-User\"]"}}
W0629 15:51:18.634396       1 authentication.go:237] Unable to get configmap/extension-apiserver-authentication in kube-system.  Usually fixed by 'kubectl create rolebinding -n kube-system ROLE_NAME --role=extension-apiserver-authentication-reader --serviceaccount=YOUR_NS:YOUR_SA'
Error: open /tmp/client-ca-file728862183: no such file or directory
Usage:
   [flags]

Flags:
      --alsologtostderr
...
panic: open /tmp/client-ca-file728862183: no such file or directory

goroutine 1 [running]:
main.main()
        /data/build/go/src/github.com/directxman12/k8s-prometheus-adapter/cmd/adapter/adapter.go:41 +0x13b
```

--- a/deploy/manifests/custom-metrics-apiserver-deployment.yaml
+++ b/deploy/manifests/custom-metrics-apiserver-deployment.yaml
@@ -39,6 +39,8 @@ spec:
         - mountPath: /etc/adapter/
           name: config
           readOnly: true
+        - mountPath: /tmp
+          name: tmp-vol
       volumes:
       - name: volume-serving-cert
         secret:
@@ -46,3 +48,5 @@ spec:
       - name: config
         configMap:
           name: adapter-config
+      - name: tmp-vol
+        emptyDir: {}

